grails-app/domain/org/pih/warehouse/inventory/CycleCount.groovy

```diff
@@ -33,7 +34,7 @@ class CycleCount {
         status = recomputeStatus()
     }
 
-    SortedSet cycleCountItems
+    SortedSet<CycleCountItem> cycleCountItems
```

Comment:
Have you made sure the items are still sorted properly? When I was helping Sebastian with that, initially I had this Set typed, but if I remember correctly the sorting was not working. In the docs they do not set type for `SortedSet`, I know this should be working with the type, but maybe it's Grails' bug that it doesn't. 
Please make sure the sorting is still working when adding the type.